numba/tests/test_unicode_array.py

```diff
@@ -303,6 +303,25 @@ def test_setitem(self):
         self.assertPreciseEqual(x1, x2)
         self.assertPreciseEqual(y1, y2)
 
+        # --------------
+        x1 = np.zeros(3)
+        v1 = np.array(3.14)
+
+        exp1 = pyfunc(x1.copy(), 0, v1)
+        got1 = cfunc(x1.copy(), 0, v1)
+        self.assertPreciseEqual(exp1, got1)
+
+        exp2 = pyfunc(x1.copy(), 2, v1)
+        got2 = cfunc(x1.copy(), 2, v1)
+        self.assertPreciseEqual(exp2, got2)
+
+        # --------------
+        x2 = np.zeros(3, dtype=np.int64)
+        v2 = np.array(3, dtype=np.int64)
+        exp3 = pyfunc(x2.copy(), 0, v2)
+        got3 = cfunc(x2.copy(), 0, v2)
+        self.assertPreciseEqual(exp3, got3)
```

Comment:
Why is this here, this is testing unicode arrays?! Perhaps it ought to go in `numba.tests.test_fancy_indexing` along with the other `setitem` tests?

Reply:
Moved to a different file